src/css/app.css

```diff
@@ -175,13 +175,18 @@ body {
     margin-top: 10px;
     margin-bottom: 60px;
 }
-.question {
+.question, .subquestion {
     font-family: "Roboto Condensed", sans-serif;
     color: #333;
     font-size: 1.1rem;
     font-weight: 400;
     margin-top: 12px;
     margin-bottom: 7px;
+    line-height: normal;
+}
+.subquestion {
+    font-size: 0.9rem;
+    margin: 0px;
```

Comment:
I prefer to have two separate, complete class definitions with the same keys but different values.

Additionally, should we define a `prose` color variable in `quasar.conf.js`, it seems that we're using `#333` for all the prose (we already have a variable `dark: #333` but I'd like to not reuse that because we use that in elements that are semantically different)

Reply:
Done